### PR TITLE
Fix/UI update

### PR DESCRIPTION
**Pull request by Longfei-Zhao**
- Update labels & Icons
- Remove wallet graphic
- Add sending & receiving the amount & Icon
- Add a tooltip for Icons
- Add outputAmount in gatewaySlice to manage receiving amount
- Add setAmount in GatewayFeesStep to manage the amount input

### CODE REVIEW

File: src/features/gateway/steps/shared/WalletSwitchHelpers.tsx [hunk 1]
```diff
@@ -171,50 +177,92 @@ const useAccountWrapperStyles = makeStyles((theme) => ({
...
+        </Typography>
+      )}
     </div>
   );
 };
 
 export const H2HAccountsResolver: FunctionComponent<
   H2HAccountsResolverProps
 > = ({ transactionType, from, to, disabled, onResolved }) => {
+  const { asset, amount, outputAmount } = useSelector($gateway);
```
[the remark this answers is not in the document]
Comment (xnyl) [reply src/features/gateway/steps/shared/WalletSwitchHelpers.tsx:246]: or locally: http://localhost:3000/release/gateway-h2h?amount=1.1&asset=DAI&from=BinanceSmartChain&to=Ethereum I suggest extracting sending/receiving components to a separate one and pass data as props.

**Review by xnyl — COMMENTED** [review]
Fix linter issues and refactor Sending/Receiving components to not use gatewaySlice as a source of truth.

File: src/features/gateway/steps/shared/WalletSwitchHelpers.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (xnyl) [reply src/features/gateway/steps/shared/WalletSwitchHelpers.tsx:246]: outupAmount should be calculated from fees derived from `(asset, amount, from, to)` params

**Comment by Longfei-Zhao** [comment]
- Remove outputAmount in gatewaySlice
- Add SendingReceivingWrapper for sending/receiving section
- Use SendingReceivingWrapper in Mint component
- Update UI: divider, padding
- Add onClose for switchWalletModal

**Pull request merged**